e2e-tests/cypress/tests/integration/channels/enterprise/system_console/channel_moderation/helpers.js

```diff
@@ -243,6 +243,12 @@ export const enableDisableAllChannelModeratedPermissionsViaAPI = (channel, enabl
                             guests: enable,
                         },
                     },
+                    {
```

Comment:
e2e tests cannot cover the full use case if we don't have a UI to check them against. I've created https://mattermost.atlassian.net/browse/MM-56198 to write them after some components of the UI are implemented